esphome/components/graphical_display_menu/graphical_display_menu.cpp

```diff
@@ -128,41 +129,48 @@ void GraphicalDisplayMenu::draw_menu_internal_(display::Display *display, const
     }
   }
 
-  int y_offset = bounds->y;
+  // Determine what items to draw
   int first_item_index = 0;
-  int last_item_index = this->displayed_item_->items_size();
+  int last_item_index = max_item_index;
```

Comment:
Is this right? You've lost the -1 offset with the switch to referencing `max_item_index`?

Reply:
That's because you're reviewing all commits at once. If you inspect the individual commits, you'll see this is a combination of a fix adding the -1 and another commit replacing it with max_item_index.

I did individual commits to avoid this type of confusion.

Reply:
No worries - was just a quick eyeball review when I got a notification on it. Will run some tests around it later today.

Reply:
It's just (another) github quirk. It show the same review comment on 2 different diffs: once on the whole PR and once on the individual commit. I can't see on which one you've commented, but I advise to look at the individual commits as those changes are accompanied by a description of what it does.